index.js

```diff
@@ -143,15 +142,15 @@ if ((os.platform() === 'win32' && require('is-administrator')()) || process.getu
             }
 
             if (program.set && program.set.length > 0) {
-                for (var i = 0; i < program.set.length; i++) {
-                    var kvPair = utils.split(program.set[i], '=', false);
+                program.set.forEach((setOption) => {
+                    const kvPair = setOption.split('=');
                     if (kvPair.length === 2) {
                         parsedConfig[kvPair[0]] = kvPair[1];
-                        saveMessages.push('Config variable: ' + kvPair[0] + ' = ' + kvPair[1] + ' been saved to config.');
+                        saveMessages.push(`Config variable: ${kvPair[0]} = ${kvPair[1]} been saved to config.`);
```

Comment:
While we're here, format should probably be "Config variable x = y has been saved to config." (add "has").

Reply:
Done!